rundeckapp/grails-spa/packages/ui-trellis/src/library/components/plugins/PluginDetails.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div>
+    <span
+        :class="descriptionCss"
+        v-if="showDescription"
+        style="margin-left: 5px"
+    >
+      {{ shortDescription }}
+    </span>
+    <details
+        class="more-info details-reset"
+        :class="extendedCss"
+        v-if="showDescription && showExtended && extraDescription"
+    >
+      <summary>
+        More...
```

Comment:
Should this text be a translatable one?

Reply:
I was trying to avoid doing structural changes to reduce the risk of breaking something  (as this string wasn't translated before I split the components), but I agree